lib/truffleruby-tool/lib/truffle_tool.rb

```diff
@@ -253,7 +253,7 @@ module OptionBlocks
     shared_offline_options = {
         offline:          ['--[no-]offline', 'Use local gems only', STORE_NEW_VALUE, false],
         offline_gem_path: ['--offline-gem-path', 'Path to a local pre-installed gems', STORE_NEW_VALUE,
-                           TRUFFLERUBY_PATH.join('truffleruby-gem-test-pack', 'gems')] }
+                           TRUFFLERUBY_PATH.join('truffleruby-gem-test-pack-3', 'gems')] } # TODO
```

Comment:
ah, I think `require_relative` might be the best option. That should work regardless of how the tool is executed.